examples/plot_sparse_recovery.py

```diff
@@ -69,11 +69,13 @@
 l0 = {}
 mse_ref = mean_squared_error(np.zeros_like(y_test), y_test)
 
+solver = AcceleratedCD(ws_strategy="fixpoint", fit_intercept=False)
+
 for idx, estimator in enumerate(penalties.keys()):
     print(f'Running {estimator}...')
-    estimator_path = cd_solver_path(
+    estimator_path = solver.path(
         X, y, compiled_clone(datafit), compiled_clone(penalties[estimator]),
-        alphas=alphas, ws_strategy="fixpoint")
```

Comment:
I believe we should keep this line to make the diff as simple as possible

Reply:
path does not accept `ws_strategy`, I pass it when I initialize the solver L72.

Reply:
my mistake, thanks!